### DISCUSSION TITLE
PWAsForFirefox 2.5.0

### DISCUSSION THREAD

**Discussion by filips123** [filips123/PWAsForFirefox#317] [Announcements]
## Added

* **An option to configure launching a web app (#265, #291):**
  An existing `firefoxpwa.openInExistingWindow` preference has been replaced to allow further configuration of how the web app should be launched when it is already opened.
  It is now possible to open a new window (default behavior), open a new tab in the existing window (behavior of previous preference), replace an existing tab, or just focus the window.

## Changed

* Updated dependencies.

## Fixed

* **Web apps from the same profile should no longer be merged on Linux (#80):**
  This release adds required code changes on the PWAsForFirefox part to fix the long-standing issue #80.
  The fix also requires some changes on the Firefox part, which will be released in Firefox 112.

* Link targets will no longer change when the out-of-scope preference changes.
* Custom web app icons should now be applied properly.
* Mute toolbar widget icon should now display properly.
* Installing extensions should now work properly (#297).
* Extensions widget should now work properly (#308).
* Permissions for runtime autoconfig files have been fixed (#311).

<hr /><em>This discussion was created from the release <a href='https://github.com/filips123/PWAsForFirefox/releases/tag/v2.5.0'>PWAsForFirefox 2.5.0</a>.</em>

**Comment by ThijmenDam** [comment]
Awesome!! I was really looking forward to this update, thanks for the effort you put into this tool. Makes my life a lot easier 😃
